wallets/shared/src/helpers.ts

```diff
@@ -1,22 +1,59 @@
-import type { EvmBlockchainMeta } from 'rango-types';
-import {
-  EvmNetworksChainInfo,
+import type {
   AddEthereumChainParameter,
-  Network,
-  Networks,
   Connect,
-  Wallet,
+  EvmNetworksChainInfo,
   InstallObjects,
+  Network,
+  Wallet,
 } from './rango';
+import type {
+  StdBlockchainInfo,
```

Comment:
This comment should be written on `rango-types`, but as we talked I will put it here.

Is there any standard for what we need here? or it's only our convention and how we structure the code? 
if there is no actual standard, I suggest to don't use `Std` prefix since `std` can be misleading for people.
`BlockchainInfo` would works for what we do here.